### ISSUE TITLE
NoPermissions (FileSystemError): Error: EACCES: permission denied

### ISSUE THREAD

**Issue by iRoberto83** [PrestaShop/docker#258]
Hi, I'm using this container with wsl and vscode.
I can't edit any of html files because I got errors like:
`Failed to save 'content.tpl': Unable to write file 'vscode-remote://wsl+ubuntu/home/roberto/sites/prestashop/html/admin123/themes/default/template/controllers/login/content.tpl' (NoPermissions (FileSystemError): Error: EACCES: permission denied, open '/home/roberto/sites/prestashop/html/admin123/themes/default/template/controllers/login/content.tpl')`
I've also tried to edit permissions with 
`sudo chown -R roberto html`
but prestashop stopped to work until I've rolled back with:
`sudo chown -R www-data html`

That's my docker-compose file:

version: "3.9"
services:
  prestashop:
    image: prestashop/prestashop:1.7-7.3-apache
    depends_on:
      - mysql
    volumes:
      - ./html:/var/www/html:rw
      - ./modules:/var/www/html/modules:rw
      - ./themes:/var/www/html/themes:rw
    ports:
      - 80:80
    environment:
      PS_DEV_MODE: "true"
      DB_SERVER: mysqldb
      DB_NAME: "${MYSQL_DATABASE}"
      DB_USER: "${MYSQL_USER}"
      DB_PASSWD: "${MYSQL_PASSWORD}"
  mysql:
    image: mysql:5.7
    restart: always
    volumes:
      - mysql-data:/var/lib/mysql
    environment:
      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
      MYSQL_DATABASE: ${MYSQL_DATABASE}
      MYSQL_USER: ${MYSQL_USER}
      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
  phpmyadmin:
    image: phpmyadmin
    restart: always
    depends_on:
      - mysql
    ports:
      - 8080:80
    environment:
      - PMA_HOST=mysql
      - PMA_USER=root
      - PMA_PASSWORD=${MYSQL_ROOT_PASSWORD}
      - UPLOAD_LIMIT=2G
volumes:
  mysql-data:

**Comment by PierreRambaud** [comment]
Yes because you're using docker images the default user for apache2/nginx is www-data
If you want to make it work you need to use something like this:
https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml
https://github.com/PrestaShop/PrestaShop/tree/develop/.docker

**Comment by iRoberto83** [comment]
> 
> 
> Yes because you're using docker images the default user for apache2/nginx is www-data
> If you want to make it work you need to use something like this:
> https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml
> https://github.com/PrestaShop/PrestaShop/tree/develop/.docker

Hi @PierreRambaud , I've solved adding my user to www-data group and chmod 775 on all the files.
I'll take a look to your files for embedding in my environment.
